webapp/components/DeleteData/DeleteData.vue

```diff
@@ -8,8 +8,10 @@
       {{ $t('settings.deleteUserAccount.pleaseConfirm', { confirm: currentUser.name }) }}
     </label>
     <ds-input v-model="enableDeletionValue" />
-    <p class="notice">{{ $t('settings.deleteUserAccount.accountDescription') }}</p>
-    <label class="checkbox">
+    <p v-show="enableDeletionValue" class="notice">
+      {{ $t('settings.deleteUserAccount.accountDescription') }}
+    </p>
+    <label v-if="currentUserCounts.contributionsCount" class="checkbox">
```

Comment:
```suggestion
    <label class="checkbox">
```

Here comes the error back you fixed in your merged PR for the missing checkbox. 😬🤔